dialogflow_v2beta1/gapic/documents_client.py

```diff
@@ -574,16 +604,16 @@ def update_document(self,
                 If a dict is provided, it must be of the same form as the protobuf
                 message :class:`~google.cloud.dialogflow_v2beta1.types.FieldMask`
             retry (Optional[google.api_core.retry.Retry]):  A retry object used
-                to retry requests. If ``None`` is specified, requests will not
-                be retried.
+                to retry requests. If ``None`` is specified, requests will
+                be retried using a default configuration.
             timeout (Optional[float]): The amount of time, in seconds, to wait
                 for the request to complete. Note that if ``retry`` is
                 specified, the timeout applies to each individual attempt.
             metadata (Optional[Sequence[Tuple[str, str]]]): Additional metadata
                 that is provided to the method.
 
         Returns:
-            A :class:`~google.cloud.dialogflow_v2beta1.types.Operation` instance.
+            A :class:`~google.cloud.dialogflow_v2beta1.types._OperationFuture` instance.
```

Comment:
update_document returns _OperationFuture rather than Operation (maybe a mistake in the config?) v2beta1